docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -289,17 +289,21 @@ const unsubscribe = room.subscribe(
   root,
   (updates) => {
     for (const update of updates) {
-      switch (update.type) {
+      const {
+        type, // "update" or "delete"
+        node,
+      } = update;
+      switch (node.type) {
```

Comment:
Both the "update" and the "node" have a `type` field. We confused the two in the example before.